tasks/connectors/cobaltio/lib/cobaltio_helper.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module Kenna
+  module Toolkit
+    module CobaltioHelper
+      @@assets = []
+
+      def cobalt_get_assets(api_token, org_token)
```

Comment:
might need to think about this later. for a large implementation all those assets would be stored in memory for the container engine and can grind processing to a halt

Reply:
This is a fair concern, though for most users the request to the `/findings` endpoint is a larger concern.

Unfortunately the (fairly new) Cobalt API does not yet support getting a specific asset (same for findings) or a subset of assets. I'll bring this up to our API team.